### PR TITLE
Clear linting errors

### PR DESCRIPTION
**Pull request by MarcusviniciusLsantos**

### CODE REVIEW

File: components/custom-network/repositories-list.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default function RepositoriesList({ repositories, onClick }) {
       const tmpRepos = [];
 
       result.forEach((item, index) => {
-        if ((item as any).value) tmpRepos.push(savedPaths[index]);
+        if ((item as any).value) tmpRepos.push(savedPaths[index]);  // eslint-disable-line
```
Comment (moshmage) [inline components/custom-network/repositories-list.tsx:49]: missing TODO fix line

File: components/pull-request-labels.tsx [hunk 2]
```diff
@@ -71,9 +71,11 @@ function PullRequestLabels({
 
   return (
     <div
-      className={`pullrequest-labels d-flex justify-content-center align-items-center bg-${getColorLabel()}-30 rounded-pill p-1 ${
-        className || ""
-      }`}
+      className={`pullrequest-labels d-flex justify-content-center
```
Comment (moshmage) [inline components/pull-request-labels.tsx:74]: make this class be a getter,
```ts
// ...
get pullRequestLabelClass() {
  return [
    `pullrequest-labels d-flex justify-content-center`,
    `align-items-center rounded-pill p-1`,
    `bg-${getColorLabel()}-30`,
    className || "",
  ].join(' ');
}
```

and then use that getter here

File: components/transactions-list.tsx [hunk 3]
```diff
@@ -31,7 +31,9 @@ import { TransactionTypes } from "interfaces/enums/transaction-types";
 import { Transaction } from "interfaces/transaction";
 
 export default function TransactionsList({
-  onActiveTransaction = (transaction) => {}
+  setActiveTransaction
+}: {
+  setActiveTransaction: React.Dispatch<React.SetStateAction<Transaction>>
```
Comment (moshmage) [inline components/transactions-list.tsx:36]: this event (`onActiveTransactionChange`) should not have a default type of a Dispatch, as we might want to do _something else_ with it before calling a dispatcher. this should simply be a `: void` [please, rename to `onActiveTransactionChange`]

File: contexts/reducers/change-balance.ts [hunk 4]
```diff
@@ -15,7 +15,8 @@ export const ChangeBalance: ReduceAction<Partial<BalanceState> | BalanceState> =
     fn: reducer
   };
 
-export const changeBalance = (payload: Partial<BalanceState> | BalanceState): ReduceActor<Partial<BalanceState> | BalanceState> => ({
+export const changeBalance = 
+(payload: Partial<BalanceState> | BalanceState): ReduceActor<Partial<BalanceState> | BalanceState> => ({
```
Comment (moshmage) [inline contexts/reducers/change-balance.ts:19]: missing space after `=\n` add 2 spaces to this line so it's correctly indented 

you can also add type for the junctions used to reduce character count,

```ts
type PayloadParam = Partial<BalanceState> | BalanceState;

export const changeBalance = (payload: PayloadParam): ReduceActor<PayloadParam> => // ... rest of code
```

File: contexts/reducers/change-bepro-init-state.ts [hunk 5]
```diff
@@ -12,4 +12,5 @@ export const ChangeBeproInit: ReduceAction<boolean> = {
   fn: reducer
 };
 
-export const changeBeproInitState = (payload: boolean): ReduceActor<boolean> => ({ name: ReduceActionName.BeproInit, payload });
+export const changeBeproInitState = (payload: boolean): ReduceActor<boolean> => 
+({ name: ReduceActionName.BeproInit, payload });
```
Comment (moshmage) [inline contexts/reducers/change-bepro-init-state.ts:16]: missing indent (lint should have caught this)

File: contexts/reducers/change-my-issues.ts [hunk 6]
```diff
@@ -10,4 +10,5 @@ export const ChangeMyIssuesState: ReduceAction<number[]> = {
   fn: reducer
 };
 
-export const changeMyIssuesState = (payload: number[]): ReduceActor<number[]> => ({ name: ReduceActionName.MyIssues, payload });
+export const changeMyIssuesState = (payload: number[]): ReduceActor<number[]> => 
+({ name: ReduceActionName.MyIssues, payload });
```
Comment (moshmage) [inline contexts/reducers/change-my-issues.ts:14]: indent

File: contexts/reducers/change-oracles.ts [hunk 7]
```diff
@@ -20,10 +20,12 @@ export const changeOraclesParse = (currentAddress: string,
       : { ...p, [c]: +oracles.amounts[i] };
 
   const delegatedEntries: [string, number][] = Object.entries(oracles.addresses.reduce(reduceAddresses, {}));
-  const delegatedToOthers = delegatedEntries.reduce((p, [address, value]) => (p += +value),
+  const delegatedToOthers = delegatedEntries.reduce((p, [, value]) => (p += +value),
```
Comment (moshmage) [inline contexts/reducers/change-oracles.ts:23]: break the line after the `=` instead

File: helpers/jimp-tools.ts [hunk 8]
```diff
@@ -56,7 +56,7 @@ export async function position(base: Jimp, image: Jimp, x = 0, y = 0) {
 
   return baseIMG.composite(positionIMG, x_axis, y_axis);
 }
-
+/* eslint-disable */   // TODO: Find other choice to text2png
```
Comment (moshmage) [inline helpers/jimp-tools.ts:59]: missing matching `eslint-enable`

File: x-hooks/use-api.tsx [hunk 9]
```diff
@@ -276,7 +270,8 @@ export default function useApi() {
   }
   
   async function getTotalBounties(state: string, 
-                                  networkName = publicRuntimeConfig?.networkConfig?.networkName): Promise<number> {
+
```
Comment (moshmage) [inline x-hooks/use-api.tsx:273]: empty line

File: x-hooks/use-api.tsx [hunk 10]
```diff
@@ -439,7 +434,11 @@ export default function useApi() {
       });
   }
 
-  async function uploadFiles(files: File | File[]): Promise<any[]> {
+  async function uploadFiles(files: File | File[]): Promise<{
```
Comment (moshmage) [inline x-hooks/use-api.tsx:437]: type this return before,
```ts
type FileUploadReturn = Promise<{hash: string; fileName: string; size: string;}[]>
```

and then use it

**Review by moshmage — CHANGES_REQUESTED** [review]

File: components/custom-network/token-configuration.tsx [hunk 11]
```diff
@@ -214,3 +215,4 @@ export default function TokenConfiguration({
     </Step>
   );
 }
+ /* eslint-disable */
```
Comment (moshmage) [inline components/custom-network/token-configuration.tsx:218]: enable?

File: components/wrong-network-modal.tsx [hunk 12]
```diff
@@ -51,8 +51,8 @@ export default function WrongNetworkModal({
           },
         ],
       });
-    } catch (error: any) {
-      if (error.code === 4902) {
+    } catch (error) {
```
Comment (moshmage) [inline components/wrong-network-modal.tsx:54]: you can type _this_ as `as {code?: number}` (or something that works) and then you don't have to repeat the typing

File: helpers/api/bepro-initializer.ts [hunk 13]
```diff
@@ -8,6 +8,7 @@ export default class Bepro {
   networkFactory: NetworkFactoryV2;
 
   async init(network = false, erc20 = false, factory = false) {
+    console.log(network, erc20)
```
Comment (moshmage) [inline helpers/api/bepro-initializer.ts:11]: lingering console log?

File: pages/[network]/connect-account.tsx [hunk 14]
```diff
@@ -43,7 +43,7 @@ export default function ConnectAccount() {
   const { t } = useTranslation(["common", "connect-account"]);
 
   const [isGhValid, setIsGhValid] = useState(null);
-  const [lastAddressBeforeConnect, setLastAddressBeforeConnect] = useState("");
+  //const [lastAddressBeforeConnect, setLastAddressBeforeConnect] = useState("");
```
Comment (moshmage) [inline pages/[network]/connect-account.tsx:46]: ?

**Review by moshmage — CHANGES_REQUESTED** [review]

File: pages/[network]/connect-account.tsx (hunk 14 above)
Comment (MarcusviniciusLsantos) [reply pages/[network]/connect-account.tsx:46]: is not being used. but I will improve the text and add the disable lint

File: helpers/api/bepro-initializer.ts (hunk 13 above)
Comment (MarcusviniciusLsantos) [reply helpers/api/bepro-initializer.ts:11]: it was the only solution I could think of without wasting so much time dealing with the warning that is being called but is not being used.

File: components/custom-network/token-configuration.tsx (hunk 11 above)
Comment (MarcusviniciusLsantos) [reply components/custom-network/token-configuration.tsx:218]: Yes. my inattention

File: components/wrong-network-modal.tsx (hunk 12 above)
Comment (MarcusviniciusLsantos) [reply components/wrong-network-modal.tsx:56]: this was the only way to get type, I created a type const to improve this repeatability

**Review by moshmage — APPROVED** [review]

**Pull request merged**